src/index.js

```diff
@@ -34,6 +41,21 @@ const DEFAULT_CONFIG = {
         { path: ROUTE_CONTRIBUTION_PLAN_BUNDLES, component: ContributionPlanBundlesPage },
         { path: ROUTE_CONTRIBUTION_PLAN_BUNDLE  + "/:contributionplanbundle_id?", component: ContributionPlanBundlePage },
         { path: ROUTE_CONTRIBUTION_PLAN_BUNDLE_REPLACE  + "/:contributionplanbundle_id", component: ContributionPlanBundleReplacePage }
+    ],
+    "admin.MainMenu": [
+        {
+            text: <FormattedMessage module="contributionPlan" id="contributionPlan.label" />,
+            icon: <LocalOffer />,
+            route: "/contributionPlans",
```

Comment:
Constants could be reused: `"/" + ROUTE_CONTRIBUTION_PLANS`

Reply:
Good point, I have added the reuse of constants